services/openuserjs/openuserjs-license.service.js

```diff
@@ -19,6 +20,11 @@ export default class OpenUserJSLicense extends BaseOpenUserJSService {
   static defaultBadgeData = { label: 'license' }
 
   transform(data) {
+    if (!('license' in data)) {
+      throw new InvalidResponse({
+        prettyMessage: 'license not found',
```

Comment:
There is some "grey area" in the oldest scripts before I fixed this... the TOS default is `MIT`... also the SPDX codes have changed at least twice for GPL so that makes it rather difficult to track since we don't usually edit the scripts. It is enforced as a requirement for several years to have at least a single license *(FOSS specifically)*.

Reply:
By this I assume you mean "license" always appears in the data, so there's no need to check if it's there or not?

Reply:
For the last few years yes however oldest scripts no. See https://openuserjs.org/scripts/sizzle/Textarea_Keyboard_Shortcuts/source which is the first script. Look at the homepage too to see how we handle it as best as we can.

Reply:
Thanks for your help!